src/modules/tools/endstops/Endstops.cpp

```diff
@@ -1200,6 +1200,54 @@ void Endstops::on_gcode_received(void *argument)
                 gcode->stream->printf(" will take effect next home\n");
                 break;
 
+            case 365: //M365, change endstop configs for pipette type
+                float value;
+                char pos;
+                switch(gcode->subcode) {
+                  case 0:
+                    // Homing Position
+                    for (auto &p : homing_axis) {
+                      if(gcode->has_letter(p.axis)) {
+                        pos = p.axis;
+                        value = gcode->get_value(p.axis);
+                        homing_axis[p.axis_index].homing_position = value;
+                        break;
+                      }
+                    }
+                    break;
+                  case 1:
+                    // max travel
+                    for (auto &p : homing_axis) {
+                      if(gcode->has_letter(p.axis)) {
+                        pos = p.axis;
+                        value = gcode->get_value(p.axis);
+                        homing_axis[p.axis_index].max_travel = value;
+                        break;
+                      }
+                    }
+                    break;
+                  case 2:
+                    // debounce
+                    pos = 'E';
+                    value = gcode->get_value('O');
+                    this->debounce_ms = value;
+                    break;
+                  case 3:
+                    // retract
+                    for (auto &p : homing_axis) {
+                      if(gcode->has_letter(p.axis)) {
+                        pos = p.axis;
+                        value = gcode->get_value(p.axis);
+                        homing_axis[p.axis_index].retract = value;
+                        break;
+                      }
+                    }
+                    break;
+                }
+                if(pos && value) {
```

Comment:
[nitpick] If you're gonna put a space in between `for` and `(` you probably should put one between `if` and `(`